### ISSUE TITLE
String encoding format issue

### ISSUE THREAD

**Issue by jprivillaso** [joyent/node-ldapjs#495]
Hi, 

I have a LDAP server with a jpeg photo in base64 format. When I search for the user, this specific attribute comes with a weird string format: 
`,��1�3�>��F��ԯ�]KW�t�.<I����zu�+\�s�9[Hd������(��4`

It seems that there's an encoding issue or something. 
Do you know what can I do ?

**Comment by tastypackets** [comment]
Yes you need to get the binary data from the attributes array, which is attached to the entry response.

**Comment by tastypackets** [comment]
If you have trouble let me know and I can send you some example code, but if you just console.log or use debugger tools on the entry.attributes you'll probably see what I mean. It's an array of objects with the name/key of what it is and the binary data. It has it for all the values, the entry.object is just a bunch of binary data converted to a string. I'm assuming author did it for simplicity sake, since most data from LDAP is simple strings.

**Comment by jsumners** [comment]
⚠️ This issue has been locked due to age. If you have encountered a recent
problem that seems to be covered by this issue, please open a new issue.

Please include a [minimal reproducible example](https://en.wikipedia.org/wiki/Minimal_reproducible_example)
when opening a new issue.
